### DISCUSSION TITLE
How to get coefficients?

### DISCUSSION THREAD

**Discussion by allisonwiener** [pymc-labs/pymc-marketing#429] [Q&A]
How do I get coefficients for each variable?

According to the documentation, print_summary() returns them, and mmm.trace['coef_media'].mean(axis=0) should as well, but my model is outputting different results for each of these. Why is that? Thank you!

**Comment by ricardoV94** [comment]
You have the whole fit results in `model.idata`. Here is a guide on how to work with this object (InferenceData) directly: https://python.arviz.org/en/stable/getting_started/WorkingWithInferenceData.html#working-with-inferencedata
